repository-springboot/README.md

```diff
@@ -29,4 +29,35 @@ Now you can access the repository at :
 http://localhost:8090/repository
 ```
 
-You can change the default port and context path via src/main/resources/application.properties.
\ No newline at end of file
+You can change the default port and context path via src/main/resources/application.properties.
+
+Running the repository on Heroku
+------------------------------
+Heroku requirements:
+1. Create account and log in
+2. Create app called "repository-springboot". If you want to use a different app name, change the appName value in pom.xml under the heroku-maven-plugin.
```

Comment:
Or a user can provide `-Dheroku.appName` when deploying to Heroku - I think it is better than writing to pom.xml. Wdyt?